### ISSUE TITLE
Populating bugzilla data takes a lot of time

### ISSUE THREAD

**Issue by marco-c** [mozilla/platatus#119]
I don't know what's going on, but lately building is far slower (also on Travis).

If I run `gulp` or `gulp watch` without removing the 'dist' directory first, it gets stuck at:
```
[15:39:38] Requiring external module babel-core/register
Building for development
[15:39:40] Using gulpfile ~/Documents/workspace/platatus/gulpfile.babel.js
[15:39:41] Starting 'build:root'...
[15:39:41] Starting 'build:tabzilla'...
[15:39:41] Starting 'build:status'...
[15:39:41] Starting 'build:css'...
[15:39:41] Starting 'build:js'...
[15:39:41] Finished 'build:css' after 278 ms
[15:39:41] Finished 'build:tabzilla' after 306 ms
[15:39:41] Finished 'build:js' after 356 ms
[15:39:41] Finished 'build:root' after 408 ms
```

I've been waiting a few minutes, but nothing happened.

The two issues could be related.

**Comment by marco-c** [comment]
Yes, if I comment out the `populateBugzillaData` call everything works.

**Comment by marco-c** [comment]
I guess this could be fixed by #113.

**Comment by digitarald** [comment]
Fixed in #113
